ProcessMaker/processmaker
# FOUR-16539: The title of header remains with My Requests in Cases redirection

## Issue & Reproduction Steps
The title of header remains with My Requests in Cases redirection

## Solution
- Update the title

## How to Test

1. Log in PM4
2. Click on Cases Tab
3. Check the title i the header of Chrome

## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-16539

## Code Review Checklist

ci:next
